client/components/Tasks/EditTasks.jsx

```diff
@@ -85,7 +85,7 @@ const EditTasks = ({
         console.log('new task added: ', data)
```

Comment:
rogue console.log

Reply:
Two rogue console.logs actually.  Lines 74 and 85